### PR TITLE
38 Allow focus log panel: Fix

### PR DESCRIPTION
**Pull request by rmadsen-ks**
- added support for tabbing to the log panel view.
- added support for shift-tab which cycles focus back instead of forward.

### CODE REVIEW

**Review by frederikja163 — CHANGES_REQUESTED** [review]
I would prefer if we save the menus to a list in the order we want to tab through them. Then we could also add the rest of the menus to this list. Like the toolbar.

File: OpenTAP.TUI/OpenTAP.TUI.cs [hunk 1]
```diff
@@ -53,11 +53,25 @@ public override bool ProcessKey(KeyEvent keyEvent)
 
             if (keyEvent.Key == Key.Tab || keyEvent.Key == Key.BackTab)
```
Comment (StefanHolst) [inline OpenTAP.TUI/OpenTAP.TUI.cs:54]: I think this behaviour could be achieved by just removing this code. This already happens with arrow keys

**Review by StefanHolst — CHANGES_REQUESTED** [review]
Just a minor comment

**Comment by frederikja163** [comment]
Closing as the issue is no longer considered an issue. See #38

**Pull request closed without merging**